README.md

```diff
@@ -2,24 +2,198 @@
+
+* A Member is removed from the ASC (a) upon having failed to have at least one representative attend two quarterly meetings in the prior twelve months, or (b) by vote of the other ASC members per the voting rules defined below. Attendance by a non-voting member of the organization (i.e., someone not officially designated as an alternate for voting purposes) counts towards meeting this requirement.
+
+The ASC meets quarterly, with at least one quarterly meeting each year to be held "in-person" at an agreed-upon location unless the ASC Members vote otherwise. A vote to "virtualize" the in-person meeting can only apply to that given year. Other quarterly meetings may be held by teleconference in accordance with the desires of ASC members. ASC quarterly meetings are conducted according to the following schedule:
+
+* 2nd week of January
+* 2nd week of April
+* 2nd week of July
+* 2nd week of October
+
+ASC Members may, if desired, vote to alter the timing of the meetings in a given year to accommodate Member schedules, subject to the voting rules described below.
+
+The Co-Chairs shall include links to each proposed modification to the Standard up for consideration during that meeting in the published agenda.
+
+#### Current ASC Members
+
+<div  align="center">
+<table>
+ <tr>
+  <td>Intel (Ralph Castain)*</td>
+  <td>IBM (Josh Hursey)**</td>
+  <td>Mellanox (Artem Polyakov)</td>
+ </tr>
+ <tr>
+  <td>UT-K (Aurelien Bouteiller)</td>
+  <td>LANL (Howard Pritchard)</td>
+  <td>ORNL (Thomas Naughton)</td>
+ </tr>
+ <tr>
+    <td>Stonybrook (Anthony Curtis)</td>
+    <td>Fujitsu Limited (Takahiro Kawashima)</td>
+    <td>ARM (Dirk Schubert)</td>
+</tr>
+<tr>
+    <td>Altair (Mike Karo)</td>
+    <td>RogueWave (John DelSignore)</td>
+</tr>
+</table>
+</div>
+\*  Co-Chair (even year) &nbsp;&nbsp;&nbsp;&nbsp; \** Co-Chair (odd year)
+
+### General voting rules
+
+The philosophy and goal of the ASC is to reach consensus on all, if not most, issues. In the event that full consensus cannot be reached, the ASC will use the following rules:
+
+* Each active ASC Member organization will receive one vote. All votes are required to pass with 2/3 majority. A quorum (>50%) of Members must be present (either physically or virtually). Abstentions will not be counted in the total votes when calculating whether there is a 2/3 majority.
+
+* A Member may submit a vote _in absentia_ to the ASC Co-Chairs in advance of the meeting if attendance by a representative is not possible (e.g., a quarterly meeting that falls during a national holiday period), but each ASC Member organization can only cast one vote. This prevents a Member from voting on behalf of its own entity and acting as a proxy for another member.
```

Comment:
The latter part of this is a little confusing to me.  Who ends up voting on behalf of the absent member?  The co-chair? A second representative from a different member institution?

Reply:
I'll clarify it. The short answer is: nobody votes on their behalf - their vote is simply recorded as if they were there and cast it themselves, with a notation that the vote was cast "in absentia"

Reply:
I clarified this:

> A Member may submit a vote _in absentia_ to the ASC Co-Chairs in advance of the meeting if attendance by a representative is not possible (e.g., a quarterly meeting that falls during a national holiday period), but each ASC Member organization can only cast one vote. This prevents a Member from voting on behalf of its own entity and acting as a proxy for another member. _In absentia_ votes are counted by the Co-Chairs as if the member were present (thus counting towards the meeting's quorum requirement), but are recorded as having been voted _in absentia_ in the official tally.